functions/supplier/cdc-debezium-supplier2/pom.xml

```diff
@@ -0,0 +1,250 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.springframework.cloud.fn</groupId>
+        <artifactId>spring-functions-parent</artifactId>
+        <version>4.0.0-SNAPSHOT</version>
+        <relativePath>../../spring-functions-parent/pom.xml</relativePath>
+    </parent>
+
+    <artifactId>cdc-debezium-supplier2</artifactId>
+    <name>cdc-debezium-supplier2</name>
+    <description>CDC Debezium Supplier V2</description>
+
+    <properties>
+        <java.version>17</java.version>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <version.debezium>2.1.4.Final</version.debezium>
+        <spring-cloud.version>2022.0.2</spring-cloud.version>
```

Comment:
Why is that?
I thought that Spring Cloud is a parent some way for all of our Stream applications...

Reply:
This is correct. All the java.version, project.build.sourceEncoding and the spring-cloud.version are already defined in the parent stream-applications-build. 
Will remove the duplications.

Reply:
resolved